src/ResourceManagement/AppService/FunctionAppsImpl.cs

```diff
@@ -58,7 +56,12 @@ public override async Task<IPagedCollection<IFunctionApp>> ListAsync(bool loadAl
                 Inner.ListNextAsync,
                 async (inner, cancellation) => await PopulateModelAsync(inner, cancellation),
                 loadAllPages, cancellationToken);
-            return PagedCollection<IFunctionApp, SiteInner>.CreateFromEnumerable(collection.Where(w => "functionapp".Equals(w.Inner.Kind, StringComparison.OrdinalIgnoreCase)));
+            return PagedCollection<IFunctionApp, SiteInner>.CreateFromEnumerable(collection.Where(w => w.Inner.Kind != null && w.Inner.Kind.Split(new char[] { ',' }).Contains("functionapp")));
```

Comment:
This seems duplicate

Reply:
Not sure why we remove StringComparison.OrdinalIgnoreCase here

Reply:
Thanks for point this out. I should use `FilterFunctionApp` uniformly.

For ignoreCase, I will add `ToLower` before `Split` as protection. Thought it should not happen in live.